content/usage/important-options.md

```diff
@@ -67,3 +67,10 @@ Use the specified panic strategy. That is, what the compiled program should do w
 
   - `-panic=trap`
     Do not print the panic message but instead of printing anything, it directly hits a trap instruction. This instruction varies by platform but it will result in the immediate termination of the program. It could either exit with `SIGILL` or cause a call to the `HardFault_Handler`. It can be used to reduce the size of the compiled program while keeping standard Go safety rules intact at the cost of debuggability.
+
+- `-scheduler`
+Use the specified scheduler. Without a scheduler goroutines won't work.
+
+  - `scheduler=coroutines` Use the coroutines scheduler.
+  - `scheduler=tasks` Use the tasks scheduler.
+  - `scheduler=none` None is the default value. No scheduler is being used.
```

Comment:
> None is the default value

This is not true. The default scheduler varies by platform. For example, AVR currently defaults to `none` because it has such limited memory while `coroutines` and `tasks` is used for other platforms. Normally you do not need to override the default except on AVR where you can optionally select the tasks scheduler if you want concurrency.

The tasks scheduler is a scheduler much like an RTOS available for a limited number of platforms. It currently works on AVR, baremetal ARM (Cortex-M), the ESP8266, and the ESP32. This is usually the preferred scheduler if it is available.
The coroutines scheduler is a portable scheduler based on C++ coroutines that works almost everywhere and is therefore used whenever the tasks scheduler is not available or impossible to implement such as on WebAssembly.
The none scheduler disables scheduler support, which means that goroutines and channels are not available. It can be used to reduce firmware size and RAM consumption if goroutines and channels are not needed.

Reply:
Thanks for the explanations, i'll update the pr :)

Reply:
Well tbh your explanations are pretty could, can i just copy them?

Reply:
Sure :)

Reply:
done :)